### DISCUSSION TITLE
What is the original plot and the portable plot?

### DISCUSSION THREAD

**Discussion by pouryarafieerakii** [Chia-Network/chia-blockchain#5701] [Support]
Are the plots we build in Chia wallet the original plots?
And can Connect them to the pool or not?
What is the original plot and the portable plot?
How to Build them?
If we want to connect to the original Chia pool, We have to start plotting on May 31st?

**Comment by sonpython** [comment]
Are the plots we build in Chia wallet the original plots? => yes for now
And can Connect them to the pool or not? => original plots could not be used to join `official pool`
What is the original plot and the portable plot? => original plot is kind of old format of plot. Portable plot is the new one that you can use to join pool, switch between pools or solo farming
How to Build them? => Wait for next announcement (maybe after May 31st)
If we want to connect to the original Chia pool, We have to start plotting on May 31st? => after May 31st (but not sure) Wait for next announcement on chia.net

**Reply by pouryarafieerakii** [reply]
thanks so mush.

**Comment by room101-dev** [comment]
> Are the plots we build in Chia wallet the original plots?

**Right now we are all getting 'classic plots' from chia-pos aka 'chia plot create' cmd-line <env>**

> And can Connect them to the pool or not?

**yes, for the past 3+ months you can 'mine' your plots on hpool.com the site is in english it takes about 2 minutes to setup there are videos on youtube, there are guides on medium.com, of course chia-net says that good sheep don't use hpool.com, 90% of all blocks are found every 5-15 minutes in ASIA with Hpool.com, seems like only the 'woke' in the West have been 'sheeped' in this scam**

> What is the original plot and the portable plot?

**original plot aka classic plot is the real deal, 'portable plot' is a new scam by Cohen ( the porn king of the internet aka bittorrent ) where he re-purposes classic plots into porn-plots aka 'portable-plots' where he rents your disk-space to others and pay's you 5 cents per day for each 1T., remember Amazon charges $20 for 1T, at 9.8EB Cohen would be earning  $20M/day off your hard-disk**

> How to Build them?

**classic plot build 'chia plot create -t fasttemp -2 slowtemp -d hdd'**

**Keep the old chia-pos software forever, because in the future chia-net will not longer let you build 'classic plots' so it must be in defiance just like right now where we're told "Do not run hpool.com, but we do, because some of us are not sheep', besides everybody knows that solo-mining doesn't work accept at pre-mine.**

**The software to build 'portable porn plots' has not yet been released, IMHO given the entire chia blockchain is non-functional the chia-dev has better things to do than buildup their porn fetishes.**

> If we want to connect to the original Chia pool, We have to start plotting on May 31st?

**If you want the 'classic plot' you started plotting two years ago,in the future just refuse to use the new 'port plot engine' what I have already done is fall back to version 1.3 of chia-plot, and I have quit updating.**

**Most of the reason that Cohen&Hoffman are pushing portable porn plots is to punish hpool.com for beating them to the punch-bowl. Thing is no blockchain on earth provides the 'pool' that's all on the outside, the blockchain provides an RPC interface called 'get-new-blocktemplate()' when called by a pool they start a new block. With CHIA they never released the pool interface so hpool.com wrote their own.**

**Given that hpool.com is already finding 90% of all new blocks &XCH in asia, they will never use the new chia software, so we will have chia-classic forever, and the new chia-porn which who in the hell knows when it will come out???? IMHO almost never because the chia-net team has been mis-managed since 2017.**

**Reply by yfycool** [reply]
HPOOL brought me revenue.
This is the truth.
Without HPOOL, I might never get XCH

**Comment by BrandtH22** [comment]
@pouryarafieerakii , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
